packages/php-wasm/web/src/lib/register-service-worker.ts

```diff
@@ -65,8 +66,14 @@ export async function registerServiceWorker(scope: string, scriptUrl: string) {
 
 	// Check if there's a new service worker available and, if so, enqueue
 	// the update:
-	await registration.update();
-
+	try {
+		await registration.update();
+	} catch (e) {
+		logger.warn(
```

Comment:
We know if the user is online, let's rethrow the error if they are.

Reply:
Makes sense, done [8003609](https://github.com/WordPress/wordpress-playground/pull/1535/commits/800360900967c861707d10068aae9882b4922a44)